compose/install/compose-plugin.md

```diff
@@ -0,0 +1,185 @@
+---
+description: How to install Docker Compose plugin
+keywords: compose, orchestration, install, installation, docker, documentation
+toc_max: 3
+
+title: Install Docker Compose Plugin
+---
+
+## Prerequisites
+
+* Docker Compose requires Docker Engine.
+* Docker Compose plugin requires Docker CLI.
+
+
+## Install Compose on Windows Server
+
+
+Follow these instructions if you are running the Docker daemon and client directly
+on Microsoft Windows Server and want to install Docker Compose.
+
+
+1.  Run a PowerShell as an administrator. 
+Be sure to, when asked if you want to allow this app to make changes to your device, to click **Yes**.
+    
+2.  GitHub now requires TLS1.2. In PowerShell, run the following:
+    
+    ```powershell  
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    ```
+3. Run the following command to download the current release of Compose ({{site.compose_version}}):
+
+    ```powershell
+    Invoke-WebRequest "https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\Docker\docker-compose.exe
+    ```
+
+    > **Note**
+    >
+    > On Windows Server 2019 you can add the Compose executable to `$Env:ProgramFiles\Docker`.
+     Because this directory is registered in the system `PATH`, you can run the `docker-compose --version` 
```

Comment:
Changes these two lines to match the way to install it as a plugin.

```suggestion
    > On Windows Server 2019 you can add the Compose CLI plugin to `$Env:ProgramFiles\Docker\cli-plugins`.
     Because this directory is used for Docker CLI plugin, you can run the `docker compose version` 
```